elements/pfe-footer/README.md

```diff
@@ -0,0 +1,119 @@
+# PFElements Footer Element
+
+`pfe-footer` is our standard Red Hat branded web applications footer.
+
+## Usage
+
+### Minimal Footer Bar
+
+```
+<pfe-footer role="contentinfo">
+  <section slot="pfe-footer-legal">
```

Comment:
For slots, would recommend `--` between the component name and the namespace of the slot: `pfe-footer--legal` or omit the namspacing entirely since these slot names are unlikely to be confusing when combined with other components.

Reply:
Yep. Need to update these. 👍🏼